docker/docker-compose.yaml

```diff
@@ -40,7 +40,7 @@ services:
     container_name: unstract-worker
     restart: unless-stopped
     entrypoint: .venv/bin/celery
-    command: "-A backend worker --loglevel=info -Q celery --autoscale=${WORKER_AUTOSCALE}"
+    command: "-A backend worker --loglevel=info -Q celery --autoscale=4,1"
```

Comment:
@gaya3-zipstack any reason why this was updated?

Reply:
Hey.,,,, Thank you for catching this... Checked-in by mistake. Will revert it.